doc/specifications/proposed/content_download/content_download.md

```diff
@@ -0,0 +1,44 @@
+# Content download
+
+> Author: bd@ez.no
+
+## Summary
+This feature covers downloading of Content binary files over HTTP.
+
+It covers:
+- generation of proper links from Twig as well as PHP,
+- permissions checking (the user must be allowed to read the Content)
+- streaming of the file over HTTP
+
+## Components
+
+### Twig helper
+
+> Name: ez_download_path
+> Usage: `ez_download_path( content, 'file' )`
+
+The helper will be used to generate a download link for a given Field/Content.
+
+### Controller action + route
+
+> Route: `/content/download/{fileId}/{fileName}`
```

Comment:
So, we Have a ddifferent fileId per every media field and version? Asking that remebering all the things we passed in legacy (object attribute id and version) if i'm not wrong.

Other than that, i like this :)